tutorials/scripting/evaluating_expressions.rst

```diff
@@ -0,0 +1,208 @@
+
++------------------------+-------------------------------------------------------------------------------------+
+| Operator               | Notes                                                                               |
++========================+=====================================================================================+
+| Addition ``+``         | Can also be used to concatenate strings and arrays:                                 |
+|                        | - ``"hello" + " world"`` = ``hello world``                                          |
+|                        | - ``[1, 2] + [3, 4]`` = ``[1, 2, 3, 4]``                                            |
++------------------------+-------------------------------------------------------------------------------------+
+| Subtraction (``-``)    |                                                                                     |
++------------------------+-------------------------------------------------------------------------------------+
+| Multiplication (``*``) |                                                                                     |
++------------------------+-------------------------------------------------------------------------------------+
+| Division (``/``)       | Performs and integer division if both operands are integers.                        |
+|                        | If at least one of them is a floating-point number, returns a floating-point value. |
++------------------------+-------------------------------------------------------------------------------------+
+| Modulo (``%``)         | Returns the remainder of an integer division.                                       |
++------------------------+-------------------------------------------------------------------------------------+
+
+Spaces around operators are optional. Also, keep in mind the usual
+`order of operations <https://en.wikipedia.org/wiki/Order_of_operations>`__
+applies. Use parentheses to override the order of operations if needed.
+
+All the Variant types supported in Godot can be used: integers, floating-point
+numbers, strings, arrays, dictionaries, colors, vectors, …
+
+Arrays and dictionaries can be indexed like in GDScript::
+
+    # Returns 1.
+    [1, 2][0]
+
+    # Returns 3. Negative indices can be used to count from the end of the array.
+    [1, 3][-1]
+
+    # Returns "green".
+    {"favorite_color": "green"}["favorite_color"]
+
+    # All 3 lines below return 7.0 (Vector3 is floating-point).
+    Vector3(5, 6, 7)[2]
+    Vector3(5, 6, 7)["z"]
+    Vector3(5, 6, 7).z
+
+Passing variables to an expression
+----------------------------------
+
+In addition to, you can pass variables to an expression. These variables will then
+become available in the expression's "context" and will be substituted when used
+in the expression::
+
+    var expression = Expression.new()
+    # Define the variable names first in the second parameter of `parse()`.
```

Comment:
I would add here another comment 

#In this example we use a name `x`

Just so people understand clearly that x is new defined name.